config/routes.rb

```diff
@@ -3,9 +3,16 @@
 Rails.application.routes.draw do
   root to: 'sessions#index'
 
+  require 'sidekiq/web'
+  Sidekiq::Web.set :sessions, false
+  mount Sidekiq::Web => '/sidekiq'
```

Comment:
Not sure if this is okay to have in production.

Reply:
It's a sidekiq dashboard btw if you haven't checked it yet

Reply:
yup 👍